### PR TITLE
gsoc24_Kurucz

### PR DESCRIPTION
**Pull request by minouHub**

### CODE REVIEW

**Comment by minouHub** [comment]
Currently waiting for approval of a second mentor

File: _projects/2024/radis/Kurucz_and_commonAPI.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+ - Knowledge of Pandas or other DataFrame libraries
+difficulty: medium
+issues:
+- https://github.com/HajimeKawahara/exojax/discussions/257
+- https://github.com/radis/radis/issues/474
+mentors:
+# First person in contact; mentors may change before project starts.
+# GitHub handles
+ - minouHub
+ - XXX
```
Comment (bsipocz) [inline _projects/2024/radis/Kurucz_and_commonAPI.md:18]: remove placeholder for now, follow-up PR should add second mentor

```suggestion
```

**Comment by bsipocz** [comment]
Ahh, actually I cannot add commits to this PR, I assume as it was opened from an organization's fork rather than from a personal fork.

**Comment by minouHub** [comment]
> I would still move ahead and merge this PR while waiting for the second mentor, so prospective participants will be able to see it in the project list.

Good point. Accepted your commit

**Pull request closed without merging**